tests/functional/s3-tagging.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+# coding=utf8
+
+from __future__ import print_function
+from six.moves.urllib.parse import urlencode
+from six import PY3
+
+import json
+import subprocess
+import time
+import unittest
+
+AWS = ["aws", "--endpoint-url", "http://localhost:5000", "s3api"]
+
+
+def to_str(val):
+    if PY3:
+        return val
+    return val.decode('utf8')
+
+
+def run_s3api(*params, **kwargs):
+    print(*params)
+    cmd = AWS + list(params)
+    try:
+        out = subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+        # some s3api commands return nothing and json.loads
+        # doesn't like it
+        if not len(out):
```

Comment:
`len()` is redundant. `if not out:` is sufficient. But that is not a release blocker. :slightly_smiling_face:

Reply:
Done